Formula/ccextractor.rb

```diff
@@ -1,9 +1,9 @@
 class Ccextractor < Formula
   desc "Free, GPL licensed closed caption tool"
-  homepage "https://www.ccextractor.org/"
-  url "https://github.com/CCExtractor/ccextractor/archive/v0.88.tar.gz"
+  homepage "https://ccextractor.org/"
+  url "https://github.com/CCExtractor/ccextractor/archive/refs/tags/v0.88.tar.gz"
```

Comment:
```suggestion
  url "https://github.com/CCExtractor/ccextractor/archive/v0.88.tar.gz"
```
We don't need to update this; they redirect to the same location.